nanoc/lib/nanoc/filters/sass.rb

```diff
@@ -25,11 +25,24 @@ def render(filter, rep, content, params = {})
         filename: rep.item.identifier.to_s,
         cache: false,
       )
+      css_path = options.delete(:css_path)
       sourcemap_path = options.delete(:sourcemap_path)
+      if sourcemap_path == :inline
+        sourcemap_path = Nanoc::Int::TempFilenameFactory.instance.create('sass_sourcemap')
+        inline = true
+      end
 
       engine = ::Sass::Engine.new(content, options)
       css, sourcemap = sourcemap_path ? engine.render_with_sourcemap(sourcemap_path) : engine.render
-      [css, sourcemap&.to_json(css_uri: rep.path, type: rep.path.nil? ? :inline : :auto)]
+
+      if inline
+        require 'base64'
```

Comment:
I’d move `require 'base64'` all the way to the top of the file. It’s part of Ruby’s stdlib so it is safe to require there.

Reply:
Or even better: stick it somewhere in this list: https://github.com/nanoc/nanoc/blob/master/nanoc/lib/nanoc.rb#L40-L62 (sorted alphabetically)

All of Nanoc’s generic requirements are in there.

It needs to be `require 'base64'` (`requires` is a Nanoc thing — it eventually results in a `require`, but it doesn’t execute `require` until the filter is actually used.)

Reply:
Done